src/handle-data.js

```diff
@@ -3,14 +3,20 @@ const directionMap = {
   D: 'desc',
 };
 
-export function getColumnOrder(layout) {
-  const { qColumnOrder, qDimensionInfo, qMeasureInfo } = layout.qHyperCube;
+const MAX_CELLS = 10000;
```

Comment:
how about MAX_CELLS_PER_PAGE

Reply:
cells = rows per page x columns per page

Reply:
But it is the max number of cells?